nvflare/apis/client_engine_spec.py

```diff
@@ -23,6 +23,13 @@
 
 class TaskAssignment(object):
     def __init__(self, name: str, task_id: str, data: Shareable):
+        """Init TaskAssignment.
```

Comment:
What does ``TaskAssignment`` mean?

Maybe @yhwen can help on some of these classes.